hw06-atm/src/main/java/ru/otus/exceptions/IncorrectWithdrawalValue.java

```diff
@@ -0,0 +1,9 @@
+package ru.otus.exceptions;
+
+public class IncorrectWithdrawalValue extends RuntimeException {
+    private static final String MESSAGE = "Некорректная сумма для снятия";
+
+    public IncorrectWithdrawalValue(int value) {
+        super(MESSAGE + ": " + value);
+    }
+}
```

Comment:
Здесь так и просится `String.format`

```suggestion
public class IncorrectWithdrawalValue extends RuntimeException {
    private static final String MESSAGE = "Некорректная сумма для снятия: %d";

    public IncorrectWithdrawalValue(int value) {
        super(String.format(MESSAGE, value));
    }
}
```